packages/fuchsia_ctl/lib/src/image_paver.dart

```diff
@@ -45,23 +46,23 @@ class ImagePaver {
   final Tar tar;
 
   /// The implementation to use for creating SSH keys.
-  final SshKeyManager sshKeyManager;
+  final SshKeyManagerProvider sshKeyManagerProvider;
 
   /// Paves an image (in .tgz format) to the specified device.
   ///
   /// The `imageTgzPath` must not be null. If `deviceName` is null, the
   /// first discoverable device will be used.
   Future<OperationResult> pave(
-    String imageTgzPath,
-    String deviceName, {
-    bool verbose = true,
-  }) async {
+      String imageTgzPath, String deviceName, String pubKeyPath,
+      {bool verbose = true, int timeoutMs = defaultPaveTimeoutMs}) async {
```

Comment:
add a comma after `defaultPaveTimeoutMs`, better formatting.  And this should be a `Duration` rather than an int.

Reply:
done